accounts/keystore/passphrase_test.go

```diff
@@ -54,7 +54,7 @@ func TestKeyEncryptDecrypt(t *testing.T) {
 		// Recrypt with a new password and start over
 		password += "new data appended" // nolint: gosec
 		if keyjson, err = EncryptKey(key, password, veryLightScryptN, veryLightScryptP); err != nil {
-			t.Errorf("test %d: failed to recrypt key %v", i, err)
+			t.Errorf("test %d: failed to encrypt key %v", i, err)
```

Comment:
I would vote for "re-encrypt" here (and also in the comment on line 54)

Reply:
error-messages for failing tests are very much not important. only us devs ever "consume" them. But sure, "recrypt" is strange, encrypt is better and so is re-encrypt

Reply:
Change to re-encrypt now